src/NServiceBus.RabbitMQ/DelayedDelivery/DelayInfrastructure.cs

```diff
@@ -79,16 +78,16 @@ public static StartupCheckResult CheckForInvalidSettings(SettingsHolder settings
         {
             var routingTopologySupportsDelayedDelivery = settings.GetOrDefault<bool>(SettingsKeys.RoutingTopologySupportsDelayedDelivery);
             var TimeoutManagerDisabled = settings.GetOrDefault<bool>(SettingsKeys.DisableTimeoutManager);
-            var timeoutManagerFeatureActive = settings.GetOrDefault<FeatureState>(typeof(TimeoutManager).FullName) == FeatureState.Active;
+            var externalTimeoutManagerAddress = settings.GetOrDefault<string>("NServiceBus.ExternalTimeoutManagerAddress") != null;
 
-            if (!routingTopologySupportsDelayedDelivery && (TimeoutManagerDisabled || !timeoutManagerFeatureActive))
+            if (!routingTopologySupportsDelayedDelivery && TimeoutManagerDisabled)
             {
                 return StartupCheckResult.Failed($"Cannot disable the timeout manager when the specified routing topology does not implement {nameof(ISupportDelayedDelivery)}.");
             }
 
-            if (!TimeoutManagerDisabled && !timeoutManagerFeatureActive)
+            if (routingTopologySupportsDelayedDelivery && externalTimeoutManagerAddress)
             {
-                return StartupCheckResult.Failed("The timeout manager is not active, but the transport has not been properly configured for this. Use 'EndpointConfiguration.UseTransport<RabbitMQTransport>().DelayedDelivery().DisableTimeoutManager()' to ensure delayed messages can be sent properly.");
+                return StartupCheckResult.Failed("Configuring an external timeout manager address is not supported.");
```

Comment:
Should this give some more information as to why? E.g.

> An external timeout manager address cannot be configured. The timeout manager is not being used because the routing topology supports delayed delivery.

?